plugins/teams/skills/coderabbit-adoption/coderabbit_adoption.py

```
@@ -0,0 +1,201 @@
+                        help="Fetch per-repo breakdowns with adoption percentages "
+                             "and check well-known repos for missing activity. "
+                             "Makes many extra API calls; prone to rate limiting.")
+    args = parser.parse_args()
+
+    end_date = args.end_date or datetime.now().strftime("%Y-%m-%d")
+    if args.start_date:
+        start_date = args.start_date
+    else:
+        start_date = (datetime.now() - timedelta(days=30)).strftime("%Y-%m-%d")
+
+    print(f"Querying GitHub search API for merged PRs in openshift org...", file=sys.stderr)
```

Ruff (0.15.4)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)